mlflow/tracking/client.py

```diff
@@ -157,6 +160,42 @@ def get_run(self, run_id: str) -> Run:
         """
         return self._tracking_client.get_run(run_id)
 
+    def get_parent_run(self, run_id: str) -> Optional[Run]:
```

Comment:
can we add a test to `tests/tracking/test_client.py`? as currently written the client and fluent APIs are the same, but would be good to keep this test coverage anyway